rotkehlchen/tests/exchanges/test_coinbase.py

```diff
@@ -494,17 +506,6 @@ def test_coinbase_query_history_events(
         asset=A_ETH,
         balance=Balance(FVal('0.00021')),
         is_fee=True,
-    ), HistoryEvent(
```

Comment:
Have you tested this change with a live coinbase account? I am not going to accept a change that removes earn events.  This seems wrong.